### ISSUE TITLE
false positive for NoPrefixForInterfacesRuleTest

### ISSUE THREAD

**Issue by paul58914080** [societe-generale/arch-unit-build-plugin-core#61]
## Summary

There seems to be an issue with the version **`2.9.1`** where even if we do not have a class prefixed with I still we get 

```log
ArchUnit Maven plugin reported architecture failures listed below :Rule Violated - com.societegenerale.commons.plugin.rules.NoPrefixForInterfacesRuleTest
[ERROR] java.lang.AssertionError: Rule 'classes that are interfaces should not be prefixed with I - this is a .Net convention.' failed to check any classes. This means either that no classes have been passed to the rule at all, or that no classes passed to the rule matched the `that()` clause. To allow rules being evaluated without checking any classes you can either use `ArchRule.allowEmptyShould(true)` on a single rule or set the configuration property `archRule.failOnEmptyShould = false` to change the behavior globally.
```

## Type of Issue

It is a : 
 - [x] bug
 - [ ] request
 - [ ] question regarding the documentation

## Current Behavior

Fails the build 

## Expected Behavior

Should build successfully 

## Steps to Reproduce (for bugs)

https://github.com/devs-from-matrix/hexagonal-spring-boot-java/pull/231

## Your Environment

* Version used: 2.9.1
* OS and version: Mac 11.5.1

**Comment by vincent-fuchs** [comment]
Thanks for raising this.. I really have no clue what happened, because the bug you report was there in 2.9.0, but was supposed to be fixed in 2.9.1 (see https://github.com/societe-generale/arch-unit-build-plugin-core/commit/5b9ef3a2a126e4a7369038a92ba3e9a5d63111ff) .

I downloaded the 2.9.1 artifact and indeed, the fix is not supposed to be there.. maybe I screwed up the release process, since I am also in the process of moving to GitHub actions. 

 Anyway, I have now published a v2.9.2 which from what I can see on the Nexus repository, now contains the expected code. 

It may not be available yet publicly, but it should, once the Maven central replication is complete. 

Can you please try to upgrade once it's available, and let me know ?

Thanks

**Comment by paul58914080** [comment]
Thanks. It is fixed now. https://github.com/devs-from-matrix/hexagonal-spring-boot-java/pull/233
